pytest_splunk_addon_ui_smartx/base_test.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         debug=False,
         cred=("admin", "Chang3d!"),
         headless=False,
+        saucelabs=False,
```

Comment:
`saucelabs` variable is not being used in this PR? should we remove it?

Reply:
removed